source/uwp/Renderer/lib/ElementTagContent.h

```diff
@@ -21,17 +23,21 @@ namespace AdaptiveNamespace
         HRESULT RuntimeClassInitialize(_In_ ABI::AdaptiveNamespace::IAdaptiveCardElement* cardElement,
                                        _In_ ABI::Windows::UI::Xaml::Controls::IPanel* parentPanel,
                                        _In_ ABI::Windows::UI::Xaml::IUIElement* separator,
-                                       _In_ ABI::Windows::UI::Xaml::Controls::IColumnDefinition* columnDefinition);
+                                       _In_ ABI::Windows::UI::Xaml::Controls::IColumnDefinition* columnDefinition,
+                                       _In_ boolean expectedVisibility);
 
         virtual HRESULT get_ColumnDefinition(_COM_Outptr_ ABI::Windows::UI::Xaml::Controls::IColumnDefinition** columnDefinition) override;
         virtual HRESULT get_AdaptiveCardElement(_COM_Outptr_ ABI::AdaptiveNamespace::IAdaptiveCardElement** cardElement) override;
         virtual HRESULT get_Separator(_COM_Outptr_ ABI::Windows::UI::Xaml::IUIElement** separator) override;
         virtual HRESULT get_ParentPanel(_COM_Outptr_ ABI::Windows::UI::Xaml::Controls::IPanel** parentPanel) override;
+        virtual HRESULT get_ExpectedVisibility(_Outptr_ boolean* expectedVisibility) override;
```

Comment:
>get_ExpectedVisibility [](start = 24, length = 22)

Let's stick a comment on this explaining the issue. I don't think it will be clear to future developers why this visibility is "expected" rather than just the visibility.